src/app/plugins/platform/Docker/plugins/cms/WordPress/createComposerFile.ts

```diff
@@ -63,7 +63,7 @@ function createComposerFile(name: string, documentRoot: string): ComposerFile {
 
   function createInstallerPath(path: string, type: string): InstallerPaths {
     const key = posix.join(documentRoot, 'wp-content', path, '{$name}');
-    const value = [`type: ${type}`];
+    const value = [`type:${type}`];
```

Comment:
This issue is breaking some recently-generated WordPress projects. Should we cherry-pick it so that it can be integrated separately from this PR?

Reply:
If it's breaking some active projects now let's cherry-pick it into a patch release.

Reply:
Opened: #180